services/blockchain-connector/shared/sdk/endpoints.js

```diff
@@ -249,6 +249,20 @@ const dryRunTransaction = async (transaction) => {
 	}
 };
 
+// TODO: Update endpoint when available from SDK
+const getFeeConstants = async () => {
+	try {
+		const apiClient = await getApiClient();
+		const response = {} || apiClient._channel.invoke('');
+		return response;
+	} catch (err) {
+		if (err.message.includes(timeoutMessage)) {
+			throw new TimeoutException('Request timed out when calling \'getGenerators\'.');
+		}
+		throw err;
+	}
+};
```

Comment:
This method must got in `services/blockchain-connector/shared/sdk/fee.js`.
```suggestion
const getFeeConstants = async () => {
	try {
		// TODO: Update endpoint when available from SDK
		const apiClient = await getApiClient();
		const response = apiClient._channel.invoke('');
		return response;
	} catch (err) {
		if (err.message.includes(timeoutMessage)) {
			throw new TimeoutException('Request timed out when calling \'getFeeConstants\'.');
		}
		throw err;
	}
};
```